deepinv/physics/multiscale.py

```diff
@@ -0,0 +1,137 @@
+import torch
+
+from deepinv.physics import Physics, LinearPhysics
+from deepinv.physics.blur import Upsampling
+
+
+class MultiScalePhysics(Physics):
```

Comment:
Can this not be achieved with the ComposedPhysics operators? e.g.

```python
physics_ms = physics * Physics(A=lambda x: upsample(x))
physics_ms = physics * LinearPhysics(A=lambda x: upsample(x), A_adjoint=lambda y: downsample(y))
```
These classes feel quite specific to RAM, so the helper methods `upsample(...)` etc. can be put in RAM, rather than having this class in physics.

If there is a reason not to do that, then this physics should be put in the docs + tested. 

Similar comment with Pad below.